### PR TITLE
Interactive chart generated using bokeh

### PR DESCRIPTION
**Pull request by gauravk-in**

### CODE REVIEW

**Comment by mtakahar** [comment]
1. Is bokeh a replacement of matplotlib?
2. Can you add some examples of annotating some data points with different texts and showing them with a mouse click, alt+click, etc.?

File: src/actions/reports/score.py [hunk 1]
```diff
@@ -75,6 +82,78 @@ def create_default_query_plots(self):
...
+        nonselected_circle = Circle(fill_alpha=0.2, fill_color="blue", line_alpha=0.2)
+        TOOLS = 'tap, lasso_select, box_zoom, wheel_zoom, pan, save, reset, hover'
+        TOOLTIPS = """
+            <div style="width:200px;">
+            @query
+            </div>
+        """
+
+        source = ColumnDataSource(data)
+        yb_plot = figure(x_axis_label = 'Estiamted Cost',
```
Comment (tverona1) [inline src/actions/reports/score.py:118]: typo

**Comment by gauravk-in** [comment]
@mtakahar 

> 1. Is bokeh a replacement of matplotlib?
Yes, this would be a replacement of matplotlib.

> 2. Can you add some examples of annotating some data points with different texts and showing them with a mouse click, alt+click, etc.?
I haven't found a way to have different tool tips based on different clicks yet. So far I found that it shows the tool tip when you hover on the point. Clicking the point takes you to the query. You can add more details to the tooltips.

File: src/actions/reports/score.py (hunk 1 above)
Comment (gauravk-in) [reply src/actions/reports/score.py]: Fixed.

File: src/actions/reports/score.py [hunk 2]
```diff
@@ -53,28 +61,105 @@ def calculate_score(self, query):
...
+        pg_url = '#@query_hash'
+        pg_taptool = pg_plot.select(type=TapTool)
+        pg_taptool.callback = OpenURL(url=pg_url, same_tab=True)
+        
+        hover_tool.renderers = [yb_scatter, pg_scatter]
+        
+        GRIDPLOT = gridplot([[yb_plot, pg_plot]], sizing_mode='scale_both', 
+                            merge_tools=False)
+        
+        show(GRIDPLOT)
```
Comment (qvad) [inline src/actions/reports/score.py]: Need to remove or comment this line, otherwise it might not work in jenkins

**Review by qvad — APPROVED** [review]

**Pull request merged**